README.md

```diff
@@ -3,13 +3,17 @@ this is just a test
 
 ---
 
-## H2 header 1
+## H2 header A
 
 blah blah
 
-## H2 header 2
+## H2 header B
 
 - this
 - is
 - a
 - list
+
+## H3 header
+
+[this is a link to google calendar](https://google.com/calendar)
```

Comment:
I'd rather this link to Google instead. Please fix it.

Reply:
Not sure what you mean. Can you elaborate please?

Reply:
```suggestion
[this is a link to google](https://google.com)
```

Reply:
Yup, I like it better like so too 👍